packages/vats/src/core/boot.js

```diff
@@ -137,6 +137,18 @@ const buildRootObject = (vatPowers, vatParameters) => {
         console.error('BOOTSTRAP FAILED:', e);
         throw e;
       }),
+    consumeItem: name => {
```

Comment:
To make this usable past-boot, we need to keep the root object alive (otherwise it'll get GCed, because nobody else outside this vat is holding a reference). 

If liveslots and Endo `importBundle` are doing everything correctly, then the only way to guarantee that the root object is kept alive is for cosmic-swingset to do a `controller.pinVatRoot('bootstrap')` during kernel initialization. I'll make a separate ticket to accomplish that.

(adding a top-level `let root` declaration, and changing `return Far('root', ..)` to `root = Far(..); return root;`, would mean the module namespace object keeps the root object alive, but liveslots shouldn't be retaining a reference to that namespace object: it calls `root = vatNS.buildRootObject()` and then drops the namespace, and `root` only stays alive in `slotToVal` until the kernel sends a `dispatch.dropExport`)